test/b2gedmntuples_cfg.py

```diff
@@ -18,7 +18,8 @@
                  'Number of events to process')
 
 options.register('sample',
-                 '/store/mc/RunIISpring15DR74/ZprimeToTT_M-3000_W-300_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/MINIAODSIM/Asympt50ns_MCRUN2_74_V9A-v1/80000/4EFF6C38-A6FD-E411-8194-0025905A6110.root',
+                 'file:/uscms_data/d3/skyria/Stealth_13/SPRING15/CMSSW_7_4_5_patch1/src/SIMULATION/test/Stealth13_MINIAOD.root',
```

Comment:
Hello Sal,

Maybe I am missing something. I had there the skimmedPatMuons since I am
copying the basic structure - but changing though some of the varibles. Is
n't this the same for the other Jet collections when they do
copy.deepcopy(basic) ( ie

###jets

jetsAK4 = copy.deepcopy(basic)
jetsAK4.variables += jetVars
jetsAK4.prefix = cms.untracked.string("jetAK4")
jetsAK4.src = cms.InputTag("jetUserData")
jetKeysAK4 = copy.deepcopy( jetKeys )
jetKeysAK4.jetLabel = cms.InputTag("jetUserData")

###patjets

jetsAK8 = copy.deepcopy(basic)
jetsAK8.variables += jetVars
jetsAK8.variables += jetAK8Vars
jetsAK8.prefix = cms.untracked.string("jetAK8")
jetsAK8.src = cms.InputTag("jetUserDataAK8")
jetKeysAK8 = copy.deepcopy( jetKeys )
jetKeysAK8.jetLabel = cms.InputTag("jetUserDataAK8")

 )?

What do you think is more suitable to be there?

Thanks,
Savvas

On Thu, Jul 9, 2015 at 5:23 PM, rappoccio <notifications@github.com> wrote:

> In test/b2gedmntuples_cfg.py
> <https://github.com/cmsb2g/B2GAnaFW/pull/11#discussion_r34267151>:
>
> > @@ -18,7 +18,8 @@
> >                   'Number of events to process')
> >
> >  options.register('sample',
> > -                 '/store/mc/RunIISpring15DR74/ZprimeToTT_M-3000_W-300_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/MINIAODSIM/Asympt50ns_MCRUN2_74_V9A-v1/80000/4EFF6C38-A6FD-E411-8194-0025905A6110.root',
> > +                 'file:/uscms_data/d3/skyria/Stealth_13/SPRING15/CMSSW_7_4_5_patch1/src/SIMULATION/test/Stealth13_MINIAOD.root',
>
> Let's try to avoid using private files for the tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cmsb2g/B2GAnaFW/pull/11/files#r34267151>.
>